instrumentation/reactor/reactor-3.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/reactor/HooksInstrumentation.java

```diff
@@ -19,7 +19,9 @@
 public class HooksInstrumentation implements TypeInstrumentation {
   @Override
   public ElementMatcher<TypeDescription> typeMatcher() {
-    return named("reactor.core.publisher.Hooks");
+    return named("reactor.core.publisher.Hooks")
+        // Hooks may not be loaded early enough so also match our main targets
+        .or(named("reactor.core.publisher.Flux").or(named("reactor.core.publisher.Mono")));
```

Comment:
Do `Mono` and `Flux` even have a static initializer method?

Reply:
Yeah they declare some constants. The tests fail without adding these